src/core/worker.js

```diff
@@ -355,6 +355,16 @@ var WorkerMessageHandler = {
     handler.on('GetDocRequest', function wphSetupDoc(data) {
       return WorkerMessageHandler.createDocumentHandler(data, port);
     });
+// what listener should I add here? Please suggest.
+    handler.on('', function workerCoverage() {
+      if (this.coverage) {
```

Comment:
There is no need to add a check for `this.coverage` here, because the message should only be sent by the main thread if coverage data is enabled.